src/tracing/mod.rs

```diff
@@ -404,6 +404,16 @@ impl TracingInspector {
         let gas_used =
             gas_used(context.spec_id(), interp.gas.spent(), interp.gas.refunded() as u64);
 
+        let immediate_bytes =
+            (self.config.record_immediate_bytes && op.info().immediate_size() > 0).then(|| {
+                let pc = interp.program_counter();
+                interp
+                    .bytecode
+                    .get(pc + 1..pc + 1 + op.info().immediate_size() as usize)
```

Comment:
note that this doesn't include the full `RJUMPV` immediate, which is calculated as `len = (imm[0] + 1) * 2`